test/integration/step_defs_test.go

```diff
@@ -335,7 +336,7 @@ func (f *feature) thereIsAnExportForSnapshotDir(name string) error {
 		panic(fmt.Sprintf("failed to get snapshot id for snapshot '%s', error '%v'\n", name, err))
 	}
 
-	snapshotIsiPath, err := isiClient.GetSnapshotIsiPath(ctx, f.isiPath, strconv.FormatInt(snapshotSrc.Id, 10))
+	snapshotIsiPath, err := isiClient.GetSnapshotIsiPath(ctx, f.isiPath, strconv.FormatInt(snapshotSrc.Id, 10), f.vol.VolumeContext["AccessZone"])
```

Comment:
check "f.vol.VolumeContext["AccessZone"]" is having value before accessing it.

Reply:
Checked basic flow for this function volume context will be present there as its snapshot creation and will have source volume details passed and  that will have corresponding access zone